packages/notification/src/types.ts

```diff
@@ -7,11 +7,15 @@ export type ToastNotification = {
   type: NotificationType
   title: string
   message: React.ReactNode
-  autohide: boolean
   autohideDelay?: "short" | "long"
```

Comment:
We should list this prop as deprecated too

Reply:
Good idea, I went with the same deprecated message as the autohide as it seemed to apply for both but let me know if it should be different!